x/rollup/keeper/deposits.go

```diff
@@ -24,21 +23,12 @@ func (k *Keeper) setL1BlockInfo(ctx sdk.Context, info derive.L1BlockInfo) error
 	if err != nil {
 		return types.WrapError(err, "marshal L1 block info")
 	}
-	if err = k.storeService.OpenKVStore(ctx).Set([]byte(types.KeyL1BlockInfo), infoBytes); err != nil {
-		return types.WrapError(err, "set")
+	store := k.storeService.OpenKVStore(ctx)
+	if err = store.Set([]byte(types.KeyL1BlockInfo), infoBytes); err != nil {
+		return types.WrapError(err, "set latest L1 block info")
```

Comment:
Now that `store` is only used once we should just leave it inlined instead of extracting it out into a separate var